biojava-structure/src/main/java/org/biojava/nbio/structure/io/mmtf/MmtfStructureReader.java

```diff
@@ -166,7 +168,11 @@ public void finalizeStructure() {
 	@Override
 	public void initStructure(int totalNumBonds, int totalNumAtoms, int totalNumGroups,
 			int totalNumChains, int totalNumModels, String modelId) {
```

Comment:
Here's somewhere where it might be cleaner to change the interface to `initStructure(..., PDBId)` since the (poorly named) modelId is intended to be a PDBId. The change would propogate to places that supply the modelId, like `StructureDataInterface`.

Reply:
This function implements an abstract function in the interface `StructureAdapterInterface` from `org.rcsb.mmtf.api`.
I can not modify the signature.